android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraViewManager.java

```diff
@@ -37,6 +37,11 @@ public void setType(RCTCameraView view, int type) {
         view.setCameraType(type);
     }
 
+    @ReactProp(name = "jpegQuality")
+    public void setJpegQuality(RCTCameraView view, int jpegQuality) {
+        view.setTorchMode(jpegQuality);
```

Comment:
This appears to be a mistake... 

As a side note, is it necessary to proxy all of these methods through all three java classes? I am not sure what the rationale was behind the original implementation.